src/components/List/List.test.tsx

```diff
@@ -31,21 +31,23 @@ describe("List", () => {
   ];
 
   it("renders unordered list", () => {
-    render(
+    let utils;
+    utils = render(
```

Comment:
This can be `const utils = ...` since `utils` is not reassigned.

Reply:
This is unrelated to the updates for `Heading`, but I did make this change.